apps/rule-manager/client/src/ts/components/RulesTable.tsx

```diff
@@ -202,102 +202,108 @@ const RulesTable = () => {
     rule.id === currentRuleId && {
       isSelected: true
     }, [currentRuleId])
-  
+
   const handleRefreshRules = async () => {
     await refreshRules();
     await fetchTags();
   }
 
   return <>
+    <EuiFlexGroup direction="column" gutterSize="none" style={{ height: "100%"}}>
+      <EuiFlexItem grow={0}>
```

Comment:
Is `grow={0}` the equivalent of `grow={false}`? I've been using `grow={false}` and wonder if it's preferable to use `grow={0}` for some reason. It does look more intuitive when we also have `grow={1}`, etc. for other components.